### PR TITLE
Closes #56 Added disqus comment for blog post. Contact me for the Disqus account

### LINKED ISSUE #56: comments section needed for every blog
# Acceptance Criteria:
The blog should end with comments field and users can comment

# Reference
https://www.gatsbyjs.com/blog/100days-comments/

### PR DESCRIPTION
**Pull request by viriyadhika**
## I ensure I have done the following steps

- [ ] Star the repository :star:
- [ ] Followed the [author](https://github.com/piyush97):white_check_mark:
- [ ] Tested the code properly :white_check_mark:
- [x ] Added the required feature :white_check_mark:

### CODE REVIEW

**Comment by piyush97** [comment]
image
@viriyadhika

File: src/templates/post.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ import { PageContainer } from '../components/Container';
 import { Code } from '../components/Code';
 import { Share } from '../components/Share';
 import { Seo } from '../containers/Seo';
+import { DiscussionEmbed } from 'disqus-react'
 
 export function formatReadingTime(minutes) {
   const cups = Math.round(minutes / 5);
```
Comment (piyush97) [inline src/templates/post.js:17]: You need to fix these issues too.

**Comment by viriyadhika** [comment]
If I'm understanding this correctly, the security settings from DisQus is configured to https://piyushmehta.tech. That's why it's unable to load DisQus from local server

**Comment by piyush97** [comment]
Hey @viriyadhika Could you please change the Disqus configurations to point to piyushmehta.com instead of .tech? 

Or is there a way to keep both?

**Comment by viriyadhika** [comment]
I just changed it to piyushmehta.com, hope it works! Sorry, this is the first time I work with DisQus, not very expert with it

**Comment by piyush97** [comment]
@viriyadhika before merging could you please share the screenshot of the comments component here

**Comment by viriyadhika** [comment]
import { DiscussionEmbed } from 'disqus-react'

const disqusConfig = {
    shortname: process.env.GATSBY_DISQUS_NAME,
    config: { 
      identifier: frontmatter.title,
    },
};

**<DiscussionEmbed { ...disqusConfig } />**
^This is the DisQus component

**Comment by piyush97** [comment]
Hey @viriyadhika sorry if you misunderstood but we need the screenshot of the rendered component as we can't check if it's working as required or not in the dev environment and please complete the task list too. 

:)

**Comment by piyush97** [comment]
Build has failed: https://app.netlify.com/sites/piyushmehta/deploys/5f75a8b722faa80007b7b2b8

**Review by piyush97 — CHANGES_REQUESTED** [review]
Need to pass the CI test and tasks required to get approved.

**Comment by viriyadhika** [comment]
Alright, I'll check it out again

**Comment by viriyadhika** [comment]
Hi, I'm not really sure why the deployment keeps failing. Can you assign this issue to other people? Sorry for the trouble. Thanks!

**Pull request closed without merging**